src/components/pages/course-view/CourseViewContainer.jsx

```diff
@@ -3,7 +3,11 @@ import { Card } from 'antd';
 import { ModuleView } from '../';
 import { useDispatch, useSelector } from 'react-redux';
 import { coursesActions } from '../../../state/ducks/coursesDuck';
+/*
```

Comment:
Perhaps this is extra information that you didn't want to include in this file???

Reply:
lol so funny! yup, thanks for the catch